tofu/aws/cluster_nodes/terraform.tf

```diff
@@ -0,0 +1,13 @@
+terraform {
+  required_version = ">= 0.13.1"
```

Comment:
Is this the correct opentofu version?

Reply:
Is this the version we should be using? 1.8.2

Reply:
updated the minimum version based on offline discussion